package.json

```diff
@@ -2308,15 +2308,11 @@
                     "default": "docker",
                     "description": "%vscode-docker.config.docker.dockerPath%",
                     "scope": "machine-overridable"
-                }
-            }
-        },
-        "configurationDefaults": {
-            "[dockercompose]": {
-                "editor.quickSuggestions": {
-                    "other": true,
-                    "comments": false,
-                    "strings": true
+                },
+                "docker.enableDockerComposeLanguageService": {
```

Comment:
Would it make sense to roll Compose langserv as an experiment?

I am not sure if I would keep the old completion provider etc... but maybe just default `enableDockerComposeLanguageService` to false and gradually increase the population that defaults to true.

Reply:
Yeah. I definitely considered it. I'm not counting it out, but it's tricky to do an experiment where the population can _choose_ to be in treatment or control; and I think it _is_ important for them to be able to choose on this.

Reply:
Agreed, but maybe we can treat this experiment in a special way, foremost as means to turn the default to off quickly if we detect that treatment population is running into unforeseen issues. At least at the beginning.

Reply:
I created #3208 to look into this. For now I think we should check in as-is, so we can get a build for CTI tonight.